cypress/e2e/base.spec.ts

```diff
@@ -20,4 +20,31 @@ describe('Visit directory page and verify users exists', () => {
   });
 });
 
+describe('Onboarding Page', () => {
+  afterEach(() => {
+    cy.truncateDatabase();
+  });
+
+  it('displays the correct heading', () => {
+    cy.visit('http://localhost:3000/onboarding');
+    cy.contains('h1', 'Onboarding');
+  });
+
+  it('fills out the form successfully', () => {
+    cy.visit('http://localhost:3000');
+    cy.login();
+    cy.visit('http://localhost:3000/onboarding');
+    cy.get('form').within(() => {
+      cy.get('textarea[name="bio"]').type('Lorem ipsum dolor sit amet.');
+      cy.get('input[name="twitter_username"]').type('test_twitter');
+      cy.get('input[name="linkedin_url"]').type('https://www.linkedin.com/in/test_linkedin');
+      cy.get('input[name="github_username"]').type('test_github');
+      cy.get('input[name="website"]').type('https://www.testwebsite.com');
+      cy.visit('http://localhost:3000/directory');
```

Comment:
This doesn't test that the submission works... it should `click()` the submit button, the redirect will be automatic if the form submission is a success.

Also, there's nothing testing the disabled/enabled state of the login button or the presence of the loading animation. Not sure if those are things we wish to test or if that's a bit too granular.